python/hail/expr/expression.py

```diff
@@ -326,7 +326,10 @@ def _init(self):
 
     def __nonzero__(self):
         raise NotImplementedError(
-            "The truth value of an expression is undefined\n  Hint: instead of if/else, use 'f.cond'")
+            "The truth value of an expression is undefined\n"
```

Comment:
Add period? What's your convention.

Reply:
generally no period for the first line, that's what Python does.